enderio-base/src/main/java/crazypants/enderio/base/config/config/DarkSteelConfig.java

```diff
@@ -153,6 +220,65 @@
   public static final IValue<Integer> energyUpgradeLevelCostEmpowered4 = F_ENERGY.make("upgradeCost5", 20, //
       "Cost for the 'Empowered V' upgrade in levels.").setRange(1, 99).sync();
 
+  public static final IValueFactory F_SWIM = F_UPGRADES.section(".swim");
+
+  public static final IValue<Integer> swimCost = F_SWIM.make("upgradeCost", 4, "Number of levels required for the 'Swim' upgrade.").setRange(1, 99).sync();
+
+  public static final IValueFactory F_NIGHT_VISION = F_UPGRADES.section(".nightVision");
+
+  public static final IValue<Integer> nightVisionCost = F_NIGHT_VISION.make("upgradeCost", 4, "Number of levels required for the 'Night Vision' upgrade.")
+      .setRange(1, 99).sync();
+
+  public static final IValueFactory F_TOP = F_UPGRADES.section(".theOneProbe");
+
+  public static final IValue<Integer> topCost = F_TOP.make("upgradeCost", 4, //
+      "Number of levels required for 'The One Probe' upgrade.").setRange(1, 99).sync();
+
+  public static final IValueFactory F_GLIDER = F_UPGRADES.section(".glider");
+
+  public static final IValue<Integer> gliderCost = F_GLIDER.make("upgradeCost", 4, //
+      "Number of levels required for the 'Glider' upgrade.").setRange(1, 99).sync();
+  public static final IValue<Double> gliderHorizontalSpeed = F_GLIDER.make("horizontalSpeed", 0.03, //
+      "Horizontal movement speed modifier when gliding.").setRange(0.001, 0.6).sync();
+  public static final IValue<Double> gliderVerticalSpeed = F_GLIDER.make("verticalSpeed", -0.05, //
+      "Rate of altitude loss when gliding.").setRange(-1, -0.001).sync();
+  public static final IValue<Double> gliderVerticalSpeedSprinting = F_GLIDER.make("verticalSpeedSprinting", -0.15, //
+      "Rate of altitude loss when sprinting and gliding.").setRange(-3, -0.001).sync();
+  
+  public static final IValueFactory F_ELYTRA = F_UPGRADES.section(".elytra");
+  
+  public static final IValue<Integer> elytraCost = F_ELYTRA.make("upgradeCost", 10, //
+      "Number of levels required for the 'Elytra' upgrade.").setRange(1, 99).sync();
+
+  public static final IValueFactory F_SOUND_LOCATOR = F_UPGRADES.section(".soundLocator");
+
+  public static final IValue<Integer> soundLocatorCost = F_SOUND_LOCATOR.make("upgradeCost", 4, //
+      "Number of levels required for the 'Sound Locator' upgrade.").setRange(1, 99).sync();
+  public static final IValue<Integer> soundLocatorRange = F_SOUND_LOCATOR.make("range", 40, //
+      "Range of the 'Sound Locator' upgrade.").setRange(1, 200).sync();
+  public static final IValue<Integer> soundLocatorLifespan = F_SOUND_LOCATOR.make("lifespan", 40, //
+      "Number of ticks the 'Sound Locator' icons are displayed for.").setRange(1, 200).sync();
+
+  public static final IValueFactory F_GOGGLES_OF_REVEALING = F_UPGRADES.section(".gogglesofrevealing");
+
+  public static final IValue<Integer> gogglesOfRevealingCost = F_GOGGLES_OF_REVEALING.make("gogglesOfRevealingUpgradeCost", 4, //
```

Comment:
"gogglesOfRevealingUpgradeCost" -> upgradeCost

same for robesUpgradeCost below

Reply:
PS: The string, not the field name